app/src/lib/emails/confirmRegistrationTemplate.tsx

```diff
@@ -0,0 +1,150 @@
+import { useCopyToClipboard } from "@/hooks/use-copy-to-clipboard";
+import { CityAttributes } from "@/models/City";
+import { UserAttributes } from "@/models/User";
+import React, { useState } from "react";
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Html,
+  Link,
+  Preview,
+  Text,
+  Img,
+  Font,
+  Hr,
+} from "@react-email/components";
+
+export default function ConfirmRegistrationTemplate({
+  url = "/",
+  user = { name: "" },
+}: {
+  url?: string;
+  user?: { name: string };
+}) {
+  const ImageURL = "https://citycatalyst.openearth.dev/assets/icon.png";
+  return (
+    <Html>
+      <Head>
+        <Font
+          fontFamily="Roboto"
+          fallbackFontFamily="Verdana"
+          webFont={{
+            url: "https://fonts.gstatic.com/s/roboto/v27/KFOmCnqEu92Fr1Mu4mxKKTU1Kg.woff2",
```

Comment:
This can be tricky for GDPR. We could start self-hosting the font or use a different mirror than Google Fonts perhaps (in a future PR).